README.md

```diff
@@ -13,3 +13,35 @@ Contents of this repo:
 ## License
 
 (c) Google, 2015. Licensed under [Apache-2](LICENSE) license.
+
+## Usage
+
+Optionally, if you would not like `goacme` to generate the account key for you, you should create an unencrypted private key for use with account registration. This can be done using OpenSSL:
+
+```
+mkdir -p ~/.config/acme
+openssl -out ~/.config/acme/account.key 2048
+```
```

Comment:
a nit: could you remove all triple back-quotes and just intend the blocks with 4 spaces? It'll make the readme cleaner as there's no special syntax highlighting required.